### DISCUSSION TITLE
v18.19.0 not on winget

### DISCUSSION THREAD

**Discussion by farhansolodev** [JanDeDobbeleer/oh-my-posh#4415] [Troubleshoot] [answered]
Running `winget upgrade JanDeDobbeleer.OhMyPosh -s winget` while on `v18.18.1` gives:
```
No available upgrade found.
No newer package versions are available from the configured sources.
```

**Comment by JanDeDobbeleer** [comment, marked as answer]
@farhansolodev that takes a while to go through their system. Nothing I can do about that.
